docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Getting started with Grafana Mimir using the Helm chart"
+menuTitle: "Getting started using the Helm chart"
+description: "Learn how to get started with Grafana Mimir using the Helm chart."
+weight: 20
+---
+
+# Getting started with Grafana Mimir using the Helm chart
+
+<!-- What is this about? TODO -->
+<!-- TODO introduce the notation of <...> to mean that the user needs to set their own value here. -->
+
+## Before you begin
+
+The instructions that follow are common across any flavor of Kubernetes.
+
+The information that follows assumes that you are able to install Kubernetes, and configure and operate it.
+
+Hardware requirements:
+
+- For this tutorial a single Kubernetes node with a minimum of 4 cores and 16GiB RAM should be sufficient. Note that production setups start from 3 nodes for redundancy!
+
+Install the following software:
+
+- Either a [Prometheus server](https://prometheus.io/docs/prometheus/latest/installation/) or [Grafana Agent](https://grafana.com/docs/grafana-cloud/agent/#installing-the-grafana-agent).
```

Comment:
Hmm, we should phrase this in an other way, cause we'll be giving two different ways of doing things later, so this is pretty under-defined here. Remove from here?